app/src/main/java/com/imashnake/animite/ui/elements/home/MediaSmallRow.kt

```diff
@@ -31,7 +31,8 @@ fun MediaSmallRow(mediaList: List<Any>) {
 @Composable
 fun TrendingNowAnimeSmallRow(mediaList: List<TrendingNowQuery.Medium?>) {
     LazyRow(
-        modifier = Modifier.padding(start = 12.dp)
+        horizontalArrangement = Arrangement.spacedBy(12.dp),
+        contentPadding = PaddingValues(start = 12.dp, end = 12.dp)
```

Comment:
```suggestion
        contentPadding = PaddingValues(start = 24.dp, end = 24.dp)
```

`horizontalArrangement` and `contentPadding` were the exact APIs I was looking for! Note however that this does change the layout. Each card should be spaced by `24.dp`, the same applies to the `contentPadding`.